icon_validator/rules/workflow_validators/workflow_title_validator.py

```diff
@@ -35,7 +35,7 @@ def validate_title(title, file_type):
                     raise ValidationException(f"Title contains a capitalized 'Of' when it should not in {file_type} file.")
                 elif not word[0].isupper() and not word.capitalize() in title_validation_list:
                     if not word.isnumeric():
-                        if not word.lower() == "by" or word.lower() == "of":
+                        if word.lower() not in ["by", "of", "-"]:
```

Comment:
Is there any other conjunctives we'd wanna throw in here while we're at it?

Reply:
Ah good call!!